### ISSUE TITLE
A problem about test

### ISSUE THREAD

**Issue by rickyang1114** [ZyeZhang/DaC#4]
Hello! I run your code and successfully get accuracy close to that shown in your paper. Your work is so wonderful!

However, I find that if I make a tiny modification to `VisDA/target.py`, the result will change rapidly. To be more specific, in line 148 of `VisDA/target.py`, if I change `dset_loaders["test"] = DataLoader(dsets["test"], batch_size=train_bs*3, shuffle=False, num_workers=args.worker, drop_last=False)` to `dset_loaders["test"] = DataLoader(dsets["test"], batch_size=train_bs*3, shuffle=True, num_workers=args.worker, drop_last=False)`(i.e. set `shuffle=True`), then the accuracy will be much lower. 

Is this normal or I get something wrong? I would appreciate it if my question could be answered.
